commcare_connect/opportunity/tables.py

```diff
@@ -164,10 +164,10 @@ class OpportunityPaymentTable(OrgContextTable):
 
     def render_view_payments(self, record):
         url = reverse(
-            "opportunity:user_payments_table",
-            kwargs={"org_slug": self.org_slug, "opp_id": record.opportunity.id, "pk": record.pk},
+            "opportunity:worker_list",
+            kwargs={"org_slug": self.org_slug, "opp_id": record.opportunity.id},
         )
-        return mark_safe(f'<a href="{url}">View Details</a>')
+        return mark_safe(f'<a href="{url}&active_tab=payments">View Details</a>')
```

Comment:
is the url field guaranteed to have query params? otherwise this could be an invalid url

Reply:
That was a mistake! Fixed it.